drv/gimlet-hf-server/src/main.rs

```diff
@@ -61,6 +61,14 @@ struct Config {
     pub clock: u8,
 }
 
+// There isn't a great crate to do `From` implementation so do this manually
```

Comment:
ah, i suppose the alternative is giving `drv-hf-api` a dependency on `drv_stm32h7_qspi`? which, yeah, seems sad...